### ISSUE TITLE
Accessibility-aware components (and views)

### ISSUE THREAD

**Issue by trentmwillis** [trentmwillis/ember-a11y-testing#4]
As a step toward making accessibility an integrated part of developing, components (and views potentially) should be able to able to audit themselves for potential accessibility issues (sort of like [react-a11y](https://github.com/rackt/react-a11y)).

**Comment by nathanhammond** [comment]
Issue was relevant to the older version of ember-a11y-testing.
